platform/bootstrap/bootstrap.yml

```diff
@@ -300,36 +453,69 @@ Resources:
       Path: /
       Roles:
       - Ref: ClusterRole
-  BootstrapInstance:
-    Type: AWS::EC2::Instance
-    DependsOn: PublicSubnet1
+  AutoScalingGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    DependsOn:
+      - PublicSubnet1
+      - PublicSubnet2
+      - PublicSubnet3
+    UpdatePolicy:
+      AutoScalingRollingUpdate:
+        MaxBatchSize: 1
+        MinInstancesInService: !Ref ManagerSize
+        PauseTime: PT20M
```

Comment:
20 min is maybe too much. 5 or 10 min should be enough to allow an instance to stabilize.